go.mod

```diff
@@ -1,9 +1,12 @@
 module github.com/docker/compose/v2
 
-go 1.22.10
+go 1.23.0
```

Comment:
Looks like the minimum version was bumped in this PR; I need to check if that was intentional

- https://github.com/docker/buildx/pull/2965